CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Change log
 -----------
 
+* Drop the external initramfs and only use the initramfs bundled in the kernel [Florin]
```

Comment:
Can you please add a Release line too? I merged the other 2 without noticing this. Maybe you can push a commit to the other two too (rpi and upboard).